pkgs/development/tools/ols/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "ols";
-  version = "nightly-2023-11-04";
+  version = "nightly-2024-02-09";
```

Comment:
Use the `0-unstable-<date>` format.

Reply:
ols specifically calls their nightly tag `nightly`, which is why I opted to calling it nightly instead of unstable initially. is `unstable` then still preferred?

Reply:
Yes, I would argue that `unstable` is still preferred unless upstream specifically releases a tag `nightly`, but it is not this case at all.

Reply:
@thiagokokada it does have a nightly tag: https://github.com/DanielGavin/ols/tree/nightly or is that not what you mean?

Reply:
I mean, a stable nightly tag. From what I understood this is always changing right?

Reply:
yes, it always changes

Reply:
Since the "nightly" tag is a moving target, it can't be used as a (static) tag.
Therefore it falls on the "default" case of `0-unstable-...`